backend/src/openarchiefbeheer/config/health_checks.py

```diff
@@ -122,7 +122,7 @@ class HealthCheckResponse(TypedDict):
 }
 
 
-def is_configuration_complete() -> HealthCheckResponse:
+def is_configuration_complete() -> HealthCheckResponse:  # noqa: C901
```

Comment:
This linter warning is a good one to keep in mind when designing that health check library. The puzzle is: how will my library API make this case expressible, without the perceived complexity.

I like that this code hasn't fallen prey to the knee-jerk of immediate removal of repetition in the code. Collecting some cases over time to see the real pattern is easier than adding edge-cases and parameters to premature abstractions.